templates/how-it-works.html

```diff
@@ -82,47 +82,44 @@ <h2>Physical availability zones</h2>
         <img src="{{ ASSET_SERVER_URL }}b2cec06d-maas.io-node-lifecycle.svg" alt="Node lifecycle chart" />
       </div>
-      <div class="col-7"><i></i>
-
-          <h3 class="p-heading--five">New</h3>
+      <div class="col-7">
+          <h3 class="p-heading--four">New</h3>
           <p>New machines which PXE-boot on a MAAS network will be enlisted automatically if MAAS can detect their BMC parameters. The easiest way to enlist standard <abbr>IPMI</abbr> servers is simply to PXE-boot them on the MAAS network.</p>
-
-          <h3 class="p-heading--five">Commissioning</h3>
+          <h3 class="p-heading--four">Commissioning</h3>
           <p>Detailed inventory of RAM, CPU, disks, NICs and accelerators like GPUs itemised and usable as constraints for machine selection. It is possible to run your own scripts for site-specific tasks such as firmware updates.</p>
-
-          <h3 class="p-heading--five">Ready</h3>
+          <h3 class="p-heading--four">Ready</h3>
           <p>A machine that is successfully commissioned is considered &ldquo;Ready&rdquo;. It will have configured BMC credentials (on IPMI based BMCs) for ongoing  power control, ensuring that MAAS can start or stop the machine and allocate or (re)deploy it with a fresh operating system.</p>
-
-          <h3 class="p-heading--five">Allocated</h3>
+          <h3 class="p-heading--four">Allocated</h3>
           <p>Ready machines can be allocated to users, who can configure network interface bonding and addressing, and disks, such as LVM, RAID, bcache or partitioning.</p>
-
-          <h3 class="p-heading--five">Deploying</h3>
+          <h3 class="p-heading--four">Deploying</h3>
           <p>Users then can ask MAAS to turn the machine on and install a complete server operating system from scratch without any manual intervention, configuring network interfaces, disk partitions and more.</p>
-
-          <h3 class="p-heading--five">Releasing</h3>
+          <h3 class="p-heading--four">Releasing</h3>
           <p>When a user has finished with the machine, they can release it back to the shared pool of capacity. You can ask MAAS to ensure that there is a full disk-wipe of the machine when that happens.</p>
       </div>
     </div>
 </section>
 
 <section class="p-strip--accent is-shallow">
     <div class="row u-equal-height">
-        <div class="col-10 u-vertically-center">
-            <h3 class="p-heading--four u-no-margin--bottom" style="max-width: none;">MAAS is a bare-metal server provisioning tool. It is open source and free.</h3>
+        <div class="col-10 u-vertically-center u-sv3--small">
+            <h3 class="p-heading--four u-no-margin--bottom">MAAS is a bare-metal server provisioning tool. It is open source and free.</h3>
         </div>
-        <div class="col-2">
-          <span><a aria-label="Find out how to insatll MAAS" href="/install" class="p-button--positive">Install MAAS</a></span>
+        <div class="col-2 u-vertically-center">
```

Comment:
Do both of these columns need to be vertically centered? One is always going to be the bigger one.

Reply:
Removed u-vertically-center from the col-10